src/BlockTypes/ProductQuery.php

```diff
@@ -91,6 +91,14 @@ protected function initialize() {
 	protected function enqueue_data( array $attributes = [] ) {
 		parent::enqueue_data( $attributes );
 
+		if ( version_compare( $GLOBALS['wp_version'], '6.2.2', '>' ) ) {
```

Comment:
NIT: Should we put both logic in a dedicated function and add a comment? (it would be great just the link to this PR)

Reply:
> NIT: Should we put both logic in a dedicated function and add a comment? (it would be great just the link to this PR)

Good idea! I extracted the logic to separate function, do you mind taking a look one more time?

Reply:
I noticed that we should fix the check for the WP version :D